conf/spark-env.sh.template

```diff
@@ -79,3 +80,6 @@
 # Options for beeline
 # - SPARK_BEELINE_OPTS, to set config properties only for the beeline cli (e.g. "-Dx=y")
 # - SPARK_BEELINE_MEMORY, Memory for beeline (e.g. 1000M, 2G) (Default: 1G)
+
+# Options for spark-submit
```

Comment:
Is this only for spark-submit? Seems like it's also used in Spark shell

Reply:
You are right, because `spark-shell` invokes `spark-submit`, I change the description to "for spark-submit and the command that depends on spark-submit"